src/mon/OSDMonitor.cc

```diff
@@ -5072,6 +5072,20 @@ void OSDMonitor::get_pools_health(
           detail->push_back(s);
       }
     }
+
+    if (sum.num_objects > 0 && pool.application_metadata.empty()) {
```

Comment:
OK -- so I can tag the MDS pools using the MDS map and I guess add pool name heuristics (e.g. rbd, cinder, glance, vms => rbd, *rgw* => rgw) since I can't really query for objects in the OSDs. Sounds good or did you have another idea?